doc/whats_new.rst

```diff
@@ -73,6 +75,8 @@ API
 
 - Add average_dipoles function to `hnn_core.dipole`, by `Blake Caldwell`_ in `#156 <https://github.com/jonescompneurolab/hnn-core/pull/156>`_
 
+- New API for defining external drives and biases to network, by `Christopher Bailey`_ in `#221 <https://github.com/jonescompneurolab/hnn-core/pull/221>`_
```

Comment:
what exactly is the bug here? can you be more explicit?

Reply:
No bug, this was under "API". Not sure how to be more explicit, added the :func:-links to "Changes"?

Reply:
maybe you should mention the new parameter `initialise_hnn_drives` to make it more explicit. And say that it is `False` by default.